Triggers nightly build can't compile unit tests

# Expected Behavior

Triggers nightly build should only fail if something is wrong.

# Actual Behavior

Triggers nightly build can't compile all unit tests b/c they need golang 1.14 minimum and seem to be using 1.13:

```
[unit-test] # knative.dev/pkg/test/logging
[unit-test] /workspace/pkg/mod/knative.dev/pkg@v0.0.0-20200922164940-4bf40ad82aab/test/logging/tlogger.go:185:5: o.t.Cleanup undefined (type *testing.T has no field or method Cleanup)
[unit-test] note: module requires Go 1.14
[unit-test] FAIL        github.com/tektoncd/triggers/cmd/triggerrun/cmd [build failed]
[unit-test] ?           github.com/tektoncd/triggers/cmd/webhook        [no test files]
[unit-test] ?           github.com/tektoncd/triggers/examples/event-interceptors        [no test files]
```

params to the tasks the pipeline is using:

```
 NAME               TASKREF                   RUNAFTER            TIMEOUT   CONDITIONS   PARAMS
 ∙ unit-tests       golang-test                                   ---       ---          package: github.com/tektoncd/triggers, version: 1.13
 ∙ build            golang-build                                  ---       ---          package: github.com/tektoncd/triggers, version: 1.13
```

--- a/tekton/release-pipeline.yaml
+++ b/tekton/release-pipeline.yaml
@@ -32,8 +32,6 @@ spec:
       params:
         - name: package
           value: $(params.package)
-        - name: version
-          value: "1.13"
       resources:
         inputs:
           - name: source
@@ -44,8 +42,6 @@ spec:
       params:
         - name: package
           value: $(params.package)
-        - name: version
-          value: "1.13"
       resources:
         inputs:
           - name: source

